config/initializers/sidekiq.rb

```diff
@@ -1,12 +1,20 @@
 # frozen_string_literal: true
-require_relative "redis_config"
+
+require "redis"
+
+# `Redis.current` is deprecated. We need to establish a connection in
```

Comment:
Could you remove the reference to `Redis.current` being deprecated from this comment?  I don't think it will make sense in the future without having Redis.current in the file.

Reply:
I did thanks!